Bio/motifs/__init__.py

```diff
@@ -601,11 +601,14 @@ def format(self, format_spec):
         This method is deprecated; instead of motif.format(format_spec),
         please use format(motif, format_spec).
         """
-        warnings.warn("""\
+        warnings.warn(
+            """\
```

Comment:
I think this slash is not needed.

Reply:
Actually it is (surprisingly) important if you want to avoid a leading newline in the string, see discussion on #2369.

However, in this case there is no strong reason to use triple quotes - I'll tweak this, so I'm glad you queried this.